package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@edx/mockprock",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
Update the version in [setup.py](https://github.com/edx/mockprock/blob/master/setup.py#L41) to match this as well.